DFC.ServiceTaxonomy.Editor/appsettings.json

```diff
@@ -6,20 +6,42 @@
             "Microsoft": "Information"
         }
     },
-  "Neo4j": {
-    "Endpoint": {
-      "Uri": "__Neo4jUrl__",
-      "Username": "__Neo4jUser__",
-      "Password": "__Neo4jPassword__"
-    },
-    "OwlDataGeneratorConfigModel": {
-      "DefaultLanguage": "en",
-      "NamespaceName": "NCSnamespace",
-      "NamespaceIri": "https://nationalcareers.service.gov.uk",
-      "HeaderIri": "https://nationalcareers.service.gov.uk/test/",
-      "HeaderAuthor": "National Careers Service",
-      "HeaderVersion": "0.0.1(alpha)",
-      "DescriptionLabel": "NationalCareersService-ServiceTaxonomy"
+    "Neo4j": {
+        "Endpoint": {
+            "Uri": "__Neo4jUrl__",
+            "Username": "__Neo4jUser__",
+            "Password": "__Neo4jPassword__"
+        },
+        "OwlDataGeneratorConfigModel": {
+            "DefaultLanguage": "en",
+            "NamespaceName": "NCSnamespace",
+            "NamespaceIri": "https://nationalcareers.service.gov.uk",
+            "HeaderIri": "https://nationalcareers.service.gov.uk/test/",
+            "HeaderAuthor": "National Careers Service",
+            "HeaderVersion": "0.0.1(alpha)",
+            "DescriptionLabel": "NationalCareersService-ServiceTaxonomy"
+        },
+        "GraphSyncPartSettings": {
+            "Settings": [
+                {
+                    "Name": "NCS",
+                    "BagPartContentItemRelationshipType": "",
+                    "NodeNameTransform": "$\"ncs__{ContentType}\"",
+                    "PropertyNameTransform": "$\"ncs__{Value}\"",
+                    "CreateRelationshipType": "",
+                    "IdPropertyName": "uri",
+                    "GenerateIdPropertyValue": "$\"http://nationalcareers.service.gov.uk/{ContentType.ToLowerInvariant()}/{Value}\""
+                },
+                {
+                    "Name": "ESCO",
+                    "BagPartContentItemRelationshipType": "",
+                    "NodeNameTransform": "$\"esco__{ContentType}\"",
+                    "PropertyNameTransform": "",
+                    "CreateRelationshipType": "",
+                    "IDPropertyName": "uri",
+                    "GenerateIdPropertyValue": "$\"http://data.europa.eu/esco/{ContentType.ToLowerInvariant()}/{Value}\""
```

Comment:
Good spot 😃!
Could you update this in `Service Taxonomy.recipe.json` and in the optimisation check in `GraphSyncHelper` too.
Also, do we need to add the `Name` setting to `Service Taxonomy.recipe.json`?